config/vufind/Blender.ini

```diff
@@ -0,0 +1,236 @@
+; facets:
+;genericRange[] = example_field_str
+; Any fields listed below will be treated as hierarchical facets
+; (see https://wiki.apache.org/solr/HierarchicalFaceting but note that we always
+; use a trailing slash to avoid ambiquities)
+;hierarchical[] = building
+;hierarchical[] = format
+; Sort options for hierarchical facets:
+; How hierarchical facets are sorted. Default is result count, but alternative ways
+; can be specified:
+; top = Sort the top level list alphabetically, others by result count (useful e.g.
+;       for a large number of building facets where top level is organization and
+;       second level the library branch)
+; all = Sort all levels alphabetically
+hierarchicalFacetSortOptions[building] = top
+; How hierarchical facet values are displayed in the records:
+; single = Display only the deepest level (default)
+; full   = Display full hierarchy for each entry
+;hierarchicalFacetDisplayStyles[format] = full
+; Separator used when display hierarchical facet levels (default is "/")
+;hierarchicalFacetSeparators[format] = " > "
+
+; This section is reserved for special boolean facets.  These are displayed
+; as checkboxes.  If the box is checked, the filter on the left side of the
+; equal sign is applied.  If the box is not checked, the filter is not applied.
+; The value on the right side of the equal sign is the text to display to the
+; user.  It will be run through the translation code, so be sure to update the
+; language files appropriately.
+;
+; Leave the section empty if you do not need checkbox facets.
+[CheckboxFacets]
+; blender_backend facet is special and allows limiting of results to one of the
+; actual backends:
+;blender_backend:Solr = "Items in Library"
+;blender_backend:SolrAuth = "Authors"
+;blender_backend:Primo = "Electronic Material (Ex Libris CDI)"
+;blender_backend:EDS = "Electronic Material (EBSCO EDS)"
+peer_reviewed:1 = peer_reviewed
+fulltext:1 = online_resources
+
+; These settings affect the way the [Results] facets are displayed
+; If using facets at the top of search results you have more room for text.
+[Results_Settings]
+; By default, the side facets will only show 6 facets and then the "show more"
+; button. This can get configured with the showMore settings.
+; You can use the * to set a new default setting.
+showMore[*] = 6
+; Or you can set a facet specific value by using the facet name as index.
+;showMore['format'] = 10
+
+; Number of rows of values for top facets to show above the "more" button
+top_rows = 2
+; Should we show "exclude" links for some or all of the facets? Set to * for
+; all facets, use a comma-separated list to show for some of the facets, set
+; to false or omit to disable "exclude" links.
+exclude = blender_backend
+; Should we OR together facets rather than ANDing them? Set to * for
+; all facets, use a comma-separated list to apply to some of the facets, set
+; to false or omit to disable ORed facets.
+orFacets = *
```

Comment:
Does it really make sense to default everything to orFacets? EDS throws a wrench in this because some fields always act as OR fields, and other fields always act as AND fields, and there's no way to override it. I think no matter what we do here, EDS is going to make some things behave oddly if you don't configure this to exactly match it.

Reply:
I didn't realize that, and it makes more sense to follow the usual defaults. I commented orFacets out by default and copied over the comment from EDS.ini.